docs/Changelog.md

```diff
@@ -10720,6 +10720,99 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain indices to integer types</dd>
 </dl>
 
+### <a name="GatherND-11"></a>**GatherND-11**</a>
+
+  Given `data` tensor of rank `r` >= 1, and `indices` tensor of rank `q` >= 1, this operator gathers 
+  slices of `data` into an output tensor of rank `q + r - indices_shape[-1] - 1`.
+  
+  `indices` is an q-dimensional integer tensor, best thought of as a `(q-1)`-dimensional tensor of indices into `data`, 
+  where each element defines a slice of `data`
+  
+  Some salient points about the inputs' rank and shape:
+   
+  1) r >= 1 and q >= 1 are to be honored. There is no dependency condition to be met between ranks `r` and `q`.
+  
+  2) The `indices_shape[-1]` should have a value between 1 (inclusive) and rank `r` (inclusive) 
+  
+  3) The value of `indices` need to be non-negative (i.e.) >= 0. The upper bound for each value in `indices` 
+     is the `data_shape` value along the corresponding axis (i.e.) `0 <= indices[:, :, i] < data_shape[i]`
+  
+  The output is computed as follows:
+  
+  The output tensor can be thought of as filling in the `indices` tensor with the appropriate slices of the input `data`.
+   
+  1) If `indices_shape[-1] > r` => error condition
+  
+  2) If `indices_shape[-1] == r`, since the rank of `indices` is `q`, `indices` can be thought of as a `(q-1)`-dimensional tensor
+     containing 1-D tensors of dimension `r`. Let us think of each such `r` ranked tensor as `indices_slice`. 
+     Each *scalar value* corresponding to `data[indices_slice]` is filled into the corresponding location of the `(q-1)`-dimensional tensor 
+     to form the `output` tensor (Example 1 below)
+  
+  3) If `indices_shape[-1] < r`, since the rank of `indices` is `q`, `indices` can be thought of as a `(q-1)`-dimensional tensor
+     containing 1-D tensors of dimension `< r`. Let us think of each such tensors as `indices_slice`. 
```

Comment:
>1-D tensors [](start = 16, length = 11)

1-D tensors?

Reply:
yes - plural. 1-D tensors => as there are many 1-D tensors within the (q-1)-dimensional tensor